packages/main/src/CheckBox.ts

```diff
@@ -75,6 +75,7 @@ let activeCb: CheckBox;
  * @public
  * @csspart root - Used to style the outermost wrapper of the `ui5-checkbox`
  * @csspart label - Used to style the label of the `ui5-checkbox`
+ * @csspart icon - Used to style the icon of the `ui5-checkbox` when it is in displayOnly state
```

Comment:
Hi @yanaminkova I suggest removing "when it is in displayOnly state" and set the part always, otherwise we need to call the part displayOnlyIcon. But having the part always seems better to me. Consumers will still be able to change it in displayOnly.